examples/meta-counter/README.md

```diff
@@ -0,0 +1,97 @@
+<!-- cargo-rdme start -->
+
+# Meta-Counter Example Application
+
+This application demonstrates how `cross_application_call` works in Linera. In our counter example 
+which has an `handle_application_call` implemented to handle cross-application calls. To use this application one must publish and create the `counter` application on the local Linera network
+
+# How It Works
+
+It requires the application id of the counter example as a parameter. It sends a message
+to itself with a value of `0` to increment so that no value is changed but the parameters and working is 
+checked.
```

Comment:
```suggestion
It requires the application id of the counter example as a parameter.
```

Let's not document too many implementation details that aren't user-visible anyway. This one might change soon.